frame/bridge/ethereum-bsc/src/lib.rs

```diff
@@ -354,6 +402,19 @@ pub mod pallet {
 			Ok(signers)
 		}
 
+		/// Verify single header
+		/// The header number should in the range [genesis_header.number, finalized_checkpoint.number + N]
+		/// Before the first call of verify_and_update_authority_set extrinsic, genesis_header == finalized_checkpoint
+		pub fn verify_header(header: &BSCHeader) -> DispatchResult {
+			let cfg = T::BSCConfiguration::get();
+			Self::contextless_checks(&cfg, header)?;
+			let authorities = <FinalizedAuthority<T>>::get();
+			// check signer
+			let signer = Self::recover_creator(cfg.chain_id, header)?;
+			ensure!(contains(&authorities, signer), <Error::<T>>::InvalidSigner);
```

Comment:
if the header is old. We can't verify it using the latest authority set.

Reply:
Yes. So i add a new storage to track the older authorities. At least, we can ensure your header is signed by a valid authority.
BUT there are still some corner cases. 

Authority B was qualified in round 10, but not qualified at round 11, if you call this fn with a header signed by B and it's number is in round 11, this fn will return OK, but actually, the header is illegal

Reply:
we can't restrict the header number in a range. any header should be verified when user submits. 
I think all the authorities at every epoch should be stored.

Reply:
Agree. But please note, the header's number must be greater than the genesis header, that's where we start to store the authority set.

Reply:
Yes the header older than genesis is not needed.

Reply:
please review it again.